src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -0,0 +1,59 @@
+package com.rtb.projectmanagementtool.auth;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String cookieValue;
+  public Cookie currCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "sessionUserID";
+    cookieValue = "out"; // "out" if not logged in
+  }
+
+  public void loginUser(HttpServletRequest request) {
+
+    // get all cookies from request
+    Cookie[] cookies = request.getCookies();
```

Comment:
Looks good.  You're going to be reusing this piece again later in your other methods, so it might be good to move this into it's own GetCookieFromRequest method.